tests/ShopBundle/Functional/PersonalData/PersonalDataExportXmlTest.php

```diff
@@ -43,7 +42,7 @@ public function testExportXml()
         $order = $this->createOrder($currency, $status, $country);
         $product = $this->createMock(Product::class);
         $price = new Price(1, 1);
-        $orderItem = new OrderItem($order, 'test', $price, 1, 1, 'ks', 'cat', $product);
+        $orderItem = new OrderItem($order, 'test', $price, 1, 1, OrderItem::TYPE_PRODUCT, 'ks', $product->getCatnum());
```

Comment:
this seems like incomplete commit, if i look into [shopsys/shopsys#715] i see there also OrderItemDataFactory.php in ShopBundle and i cant see it here in whole PR.

Reply:
please check all file changes from that shopsys/shopsys #715 PR